dev-workflow-ui/webContent/resources/css/workflow-ui.css

```diff
@@ -245,3 +245,36 @@ body .custom-accordion.ui-accordion .ui-accordion-header:first-child {
   padding-top: 0;
   padding-bottom: 0.75em;
 }
+
+/* Notifications */
+.notification-badge {
+  position:absolute;
+  bottom:30px;
+  font-weight:bold;
+  left:20px;
+  font-size:13px;
+  width:22px;
+  text-align:center;
+  padding:2px;
+  height:20px;
+  color:white;
+  border-radius:10px;
+  background-color:var(--ivy-primary-color);
+}
```

Comment:
place properties that affect similar things next to each other:
width, height
font-weight, font-size
bottom, left

remove padding 2px and text-align: center -> replace with display flex align center horizontally and vertically:
display: flex;
align-items: center;
justify-content: center;